components/eam/src/chemistry/yaml/cam_ndrop/f90_yaml/update_from_cldn_profile_end_yml.f90

```diff
@@ -0,0 +1,44 @@
+#ifdef YAML_NDROP
+  if(yaml%flag_print) then ! if this column exists in lchnk
+
+     !write output header
+     call write_output_header(unit_output)
+
+     !start writing data
+     
+        call write_var(unit_output,'raercol_nsav(yaml%lev_print,:)',raercol_nsav)
```

Comment:
save only one cell in outputs : `raercol_nsav(yaml%lev_print,:)`

Reply:
This has been fixed.